src/screens/Profile/Header/Metrics.tsx

```diff
@@ -54,7 +54,7 @@ export function ProfileHeaderMetrics({
         </Text>
       </InlineLinkText>
       <Text style={[a.font_bold, t.atoms.text, a.text_md]}>
-        {formatCount(profile.postsCount || 0)}{' '}
+        {formatCount(i18n, profile.postsCount || 0)}{' '}
         <Text style={[t.atoms.text_contrast_medium, a.font_normal, a.text_md]}>
           {plural(profile.postsCount || 0, {one: 'post', other: 'posts'})}
         </Text>
```

Comment:
Food for thought, I feel like there's a better way to do this (where we have different styling for count and suffix) in a way that's more i18n-friendly (doesn't assume prefix/suffix)

```
{count, plural,
  one {# <0>post</0>}
  other {# <0>posts</0>}
}
```

Problem is, I don't understand ICU MessageFormat enough to know if like, there's a way to format the `#` sigil.

Perhaps this works as an alternative?

```
{count, plural,
  one {{fixedCount} <0>post</0>}
  other {{fixedCount} <0>posts</0>}
}
```

This isn't much of a priority right now though.